include/directx/d3d12.h

```diff
@@ -2300,7 +2302,8 @@ enum D3D_SHADER_MODEL
         D3D_SHADER_MODEL_6_4	= 0x64,
         D3D_SHADER_MODEL_6_5	= 0x65,
         D3D_SHADER_MODEL_6_6	= 0x66,
-        D3D_SHADER_MODEL_6_7	= 0x67
+        D3D_SHADER_MODEL_6_7	= 0x67,
+        D3D_HIGHEST_SHADER_MODEL = D3D_SHADER_MODEL_6_7 // TODO: Pull from latest d3d12.h

```

Comment:
Probably no. And actually I don't need it any more since I've changed the process of iterating through all the shader models during initialization. I guess I will remove the extra entry after all. Thanks for the notice!

Reply:
Resolved.